README.md

```diff
@@ -83,20 +88,29 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get(req, res){
+  respond.send("index")});
+app.put(req, res){
+  respond.send("create")});
+app.post(req, res){
+  respond.send("update")});
+app.delete(req, res){
+  respond.send("delete")});
 ```
 ### Question #7
 
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
+Express backend works with Javascript. Express is the backend part of the MEAN stack. There aren't any files that exist within the file.
+Rails backend works with Ruby. Rails backend  have files made but aren't utilize until it created through the command line.
```

Comment:
Unclear here too, "Rails backend  have files made but aren't utilize until it created through the command line."  Do you mean you have to start a rails server?  You need a server for express too.